chat.go

```diff
@@ -23,12 +23,20 @@ const (
 )
 
 type chatResponseFull struct {
-	Channel   string `json:"channel"`
-	Timestamp string `json:"ts"`
-	Text      string `json:"text"`
+	Channel          string `json:"channel"`
+	Timestamp        string `json:"ts"`
+	MessageTimeStamp string `json:"message_ts"`
+	Text             string `json:"text"`
 	SlackResponse
 }
 
+func (c chatResponseFull) getMessageTimestamp() string {
```

Comment:
mind adding the notes from your issue to a comment inside this method? otherwise LGTM.
useful information for people in the future looking at this code and being like WTF.

Reply:
Sounds like a good idea @james-lawrence 
Thank you 🙇